.github/workflows/_reusable_push_to_subtree.yml

```diff
@@ -0,0 +1,45 @@
+name: Push to subtree
+
+on:
+  workflow_call:
+    inputs:
+      remote:
+        type: string
+        required: true
+        description: |
+          The remote url of the subtree repository.
+          eg: https://github.com/bonitasoft/bonita-studio.git 
+      prefix: 
+        type: string
+        required: false
+        default: community
+        description: |
+          The prefix used for the community subtree. 
+          Default is 'community'
+      reference: 
+        type: string
+        required: false
+        default: dev
+        description: |
+          The reference to push to the git subtree. 
+          Default is 'dev'. Can't be a tag.
+    secrets:
+      BONITA_CI_PAT:
+        required: true
+
+jobs:
+  push-to-subtree:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
No need to use our git-setup-action?

Reply:
No, but it is a good point, it looks like the hash of the commit is changed and the verification of the commit is lost. This is annoying... I'll try using another action using the same tool that look better (https://github.com/frankdejonge/use-subsplit-publish?tab=readme-ov-file#example-workflow)

Reply:
Can we do it with a classic git command instead of using a custom github action?

Reply:
No it is more complicated than that. It needs to download the tools...etc
I don't known what git subtree or splitsh do, but they do not make a "real" commit and it looks not possible to generate signed commits](https://github.com/splitsh/lite/issues/52)